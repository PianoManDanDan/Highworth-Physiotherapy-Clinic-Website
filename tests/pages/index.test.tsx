import React from 'react';
import { getPage } from 'next-page-tester';

describe('HomePage Page', () => {
  test('matches the snapshot', async () => {
    const { render } = await getPage({
      route: '/',
    });

    const page = render();

    expect(page).toMatchSnapshot();
  });
});
