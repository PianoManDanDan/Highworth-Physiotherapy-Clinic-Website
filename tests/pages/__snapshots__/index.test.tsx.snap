// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage Page matches the snapshot 1`] = `
Object {
  "nextRoot": <div
    id="__next"
  >
    <header>
      <div>
        <a
          href="/"
        >
          <div
            style="display:inline-block;max-width:100%;overflow:hidden;position:relative;box-sizing:border-box;margin:0"
          >
            <div
              style="box-sizing:border-box;display:block;max-width:100%"
            >
              <img
                alt=""
                aria-hidden="true"
                role="presentation"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzAiIGhlaWdodD0iMTI4IiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZlcnNpb249IjEuMSIvPg=="
                style="max-width:100%;display:block;margin:0;border:none;padding:0"
              />
            </div>
            <noscript>
              <img
                alt="Highworth Physiotherapy Clinic"
                decoding="async"
                src="/assets/logos/hpc.png"
                srcset="/_next/image?url=%2Fassets%2Flogos%2Fhpc.png&w=32&q=75 1x, /_next/image?url=%2Fassets%2Flogos%2Fhpc.png&w=64&q=75 2x"
                style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
              />
            </noscript>
            <img
              alt="Highworth Physiotherapy Clinic"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
            />
          </div>
        </a>
        <div>
          <a
            href="/"
          >
            <h1>
              Highworth Physiotherapy Clinic
            </h1>
          </a>
          <address>
            <h2>
              <a
                href="tel:+441793763814"
              >
                <svg
                  fill="none"
                  height="24"
                  stroke="currentColor"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  width="24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M22 16.92v3a2 2 0 0 1-2.18 2 19.79 19.79 0 0 1-8.63-3.07 19.5 19.5 0 0 1-6-6 19.79 19.79 0 0 1-3.07-8.67A2 2 0 0 1 4.11 2h3a2 2 0 0 1 2 1.72 12.84 12.84 0 0 0 .7 2.81 2 2 0 0 1-.45 2.11L8.09 9.91a16 16 0 0 0 6 6l1.27-1.27a2 2 0 0 1 2.11-.45 12.84 12.84 0 0 0 2.81.7A2 2 0 0 1 22 16.92z"
                  />
                </svg>
                <span>
                  01793 763814
                </span>
              </a>
            </h2>
          </address>
        </div>
      </div>
      <nav>
        <a
          href="/"
        >
          Home
        </a>
        <a
          href="/facilities"
        >
          Facilities
        </a>
        <a
          href="/physiotherapy"
        >
          Physiotherapy
        </a>
        <a
          href="/animals"
        >
          Animals
        </a>
        <a
          href="/meet-the-team"
        >
          Meet the Team
        </a>
        <a
          href="/faq"
        >
          FAQ's
        </a>
        <a
          href="/contact"
        >
          Contact Us
        </a>
      </nav>
    </header>
    <main>
      <h1>
        HOME
      </h1>
      <p>
        Highworth Physiotherapy Clinic is a private physiotherapy and sports injury clinic conveniently located on the high street in the market town of Highworth, north east of Swindon. If you think we may be able to help you, the fastest way to make an appointment or contact us is to give us a ring. You do not need a doctor's referral unless it is required by your insurance company. Two of the physiotherapists at the clinic are also trained to treat animals but if you wish us to treat your pet then your vet must agree to this in advance.
      </p>
      <p>
        Physiotherapy is the treatment of a wide variety of injuries and conditions using a range of manual techniques, advice, exercise and electrotherapy. When appropriate, a lot of time is spent teaching the patient self help exercises so that they can manage their own condition. All the physiotherapists at the practice are experienced Chartered Physiotherapists who treat sports injuries, back pain, neck pain, arthritis, aches, strains and pains. We are committed to a high level of clinical expertise and customer satisfaction.
      </p>
      <p>
        Highworth Physiotherapy clinic was started in 1989 by Sonya Nightingale. It expanded and moved to its current location in a grade II listed building in the High Street of Highworth in 2003. It is conveniently situated between a pub and a bakery for after treatment refreshments! We have three fully equipped treatment rooms including one on the ground floor for patients unable to use the stairs.
      </p>
      <h2>
        What to expect from your treatment
      </h2>
      <p>
        We usually allow 45 minutes for your initial assessment, and 30 minutes for any follow up treatments. However these times may vary depending on the condition being treated, and this time also includes time to write up your notes.
      </p>
      <p>
        At the assessment your physiotherapist will ask you questions about your condition and general health. They will carry out an examination in order to assess how you respond to treatment as some people respond better than others. Your physiotherapist will then agree a treatment plan with you and, depending on the physiotherapists findings, they may recommend further treatments. You will also be given advice on self management of your condition and may be given exercises to carry out yourself at home.
      </p>
      <p>
        It is best to wear loose fitting clothing for your treatments and your physiotherapist may require you to remove some clothing to assist in the assessment and treatment.
      </p>
      <p>
        Your physiotherapy treatment like any other medical treatments is confidential. No information about you will be divulged to anyone else except your GP or consultant, without your prior written permission.
      </p>
      <p>
        We work with several insurance providers to provide treatment to their customers. If you would like to take advantage of this, please check with your insurance provider to see if we are an approved healthcare provider.
      </p>
    </main>
    <footer>
      <div>
        <div>
          <p>
            Find us on Social Media
          </p>
          <div>
            <a
              href="https://www.facebook.com/HighworthPhysio/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <svg
                fill="none"
                height="32"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
                viewBox="0 0 24 24"
                width="32"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M18 2h-3a5 5 0 0 0-5 5v3H7v4h3v8h4v-8h3l1-4h-4V7a1 1 0 0 1 1-1h3z"
                />
              </svg>
            </a>
          </div>
        </div>
        <div>
          <p>
            All of our physios are registered with at least one of the following regulatory bodies:
          </p>
          <ul>
            <li>
              <a
                href="https://www.csp.org.uk/"
                rel="noopener noreferrer"
                target="_blank"
              >
                <abbr
                  title="Chartered Society of Physiotherapists"
                >
                  CSP
                </abbr>
              </a>
            </li>
            <li>
              <a
                href="https://www.hcpc-uk.org/"
                rel="noopener noreferrer"
                target="_blank"
              >
                <abbr
                  title="Health and Care Professions Council"
                >
                  HCPC
                </abbr>
              </a>
            </li>
            <li>
              <a
                href="https://www.physiofirst.org.uk/"
                rel="noopener noreferrer"
                target="_blank"
              >
                Physio First
              </a>
            </li>
            <li>
              <a
                href="https://www.aacp.org.uk/"
                rel="noopener noreferrer"
                target="_blank"
              >
                <abbr
                  title="Acupuncture Association of Chartered Physiotherapists"
                >
                  AACP
                </abbr>
              </a>
            </li>
            <li>
              <a
                href="https://www.acpat.org/"
                rel="noopener noreferrer"
                target="_blank"
              >
                <abbr
                  title="Association of Chartered Physiotherapists in Animal Therapy"
                >
                  ACPAT
                </abbr>
              </a>
            </li>
            <li>
              <a
                href="https://www.rampregister.org/"
                rel="noopener noreferrer"
                target="_blank"
              >
                <abbr
                  title="Register of Animal Musculoskeletal Practitioners"
                >
                  RAMP
                </abbr>
              </a>
            </li>
          </ul>
        </div>
      </div>
    </footer>
  </div>,
}
`;
